packages/chord-symbol/src/parser/chordParserFactory.js

```diff
@@ -44,7 +44,7 @@ function chordParserFactory(parserConfiguration = {}) {
 	/**
 	 * Convert an input string into an abstract chord structure
 	 * @param {String} symbol - the chord symbol candidate
-	 * @returns {Chord|Object} A chord object if the given string is successfully parsed. An object with an `error` property otherwise.
+	 * @returns {MaybeChord} A chord data object if the given string is successfully parsed. A chord parse failure object with an `error` property otherwise.
```

Comment:
you can shorten a bit the description or add a carriage return to get rid of the eslint error